### PR TITLE
301-part-1-skip-pytest-failures

### PR DESCRIPTION
**Pull request by PeopleMakeCulture**
This PR skips all currently failing tests in order to restore the github action that runs automated tests for each PR
Related to: https://github.com/microbiomedata/nmdc-runtime/issues/301

image
image

### CODE REVIEW

File: nmdc_runtime/test.Dockerfile [hunk 1]
```diff
@@ -34,4 +34,6 @@ RUN chmod +x wait-for-it.sh
 # Best practices: Prepare for C crashes.
 ENV PYTHONFAULTHANDLER=1
 
-ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest", "-x"]
\ No newline at end of file
+# disable stop after first failure to run all tests
+ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest"]
+#ENTRYPOINT [ "./wait-for-it.sh", "fastapi:8000" , "--strict" , "--timeout=300" , "--" , "pytest", "-x"]
```
Comment (eecavanna) [inline nmdc_runtime/test.Dockerfile:39]: FYI to other reviewers:
- The `-x` option told pytest to stop after the first error

Reference: https://docs.pytest.org/en/6.2.x/usage.html#stopping-after-the-first-or-n-failures

**Review by eecavanna — APPROVED** [review]
I'm OOO today.

The changes look good to me.

**Review by dwinston — DISMISSED** [review]
image

**Review by pkalita-lbl — DISMISSED** [review]

**Review by dwinston — DISMISSED** [review]

**Comment by PeopleMakeCulture** [comment]
Made one change to test.Dockerfile to force GH Action to fail after first failed test

**Review by dwinston — APPROVED** [review]

**Pull request merged**